index.html

```diff
@@ -394,11 +396,23 @@ <h4>Computation steps</h4>
                     <li>For <code>:after</code> pseudo elements, <a class="termref">User agents</a> MUST append <abbr title="Cascading Style Sheets">CSS</abbr>  textual content, without a space, to the textual content of the <code>current node</code>. </li>
                   </ul>
                 </li>
-                <li id="step2F.iii">For each child node of the <code>current node</code>:
+                <li id="step2F.iii">
+                  Determine the <code>displayed child nodes</code> of the <code>current node</code>:
                   <ol>
-                    <li id="step2F.iii.a">Set the <code>current node</code> to the child node.</li>
-                    <li id="step2F.iii.b">Compute the text alternative of the  <code>current node</code> beginning with step 2.  Set the <code>result</code> to that text alternative.</li>
-                    <li id="step2F.iii.c">Append the <code>result</code> to the <code>accumulated text</code>. </li>
+                    <li id="step2F.iii.a">If the <code>current node</code> has an attached [=shadow root=], set the <code>displayed child nodes</code> to be the child nodes of the [=shadow root=].</li>
+                    <li id="step2F.iii.b">Otherwise, if the <code>current node</code> is a [=slot=] with [=slot/assigned nodes=], set the <code>displayed child nodes</code> to be the [=slot/assigned nodes=] of the <code>current node</code>.</li>
```

Comment:
```suggestion
                    <li id="step2F.iii.a">If the <code>current node</code> has an attached [=shadow root=], set the <code>rendered child nodes</code> to be the child nodes of the [=shadow root=].</li>
                    <li id="step2F.iii.b">Otherwise, if the <code>current node</code> is a [=slot=] with [=slot/assigned nodes=], set the <code>rendered child nodes</code> to be the [=slot/assigned nodes=] of the <code>current node</code>.</li>
```

Reply:
"set" may also be ambiguous? Is it possible someone may misinterpret this means promoting the shadow root to be in the same doc root? I doubt it, but trying to anticipate alternate readings.

Reply:
I'm using "set" in the sense of "set the variable named `renderedChildNodes` to be this value." Elsewhere in the spec it seems to use "set" in the sense of setting a variable as well.

Is there a better way I could factor this to be less ambiguous? I'm a bit unfamiliar with spec writing. 🙂